fab

```diff
@@ -0,0 +1,19 @@
+#! /bin/bash
+
+ENV=.deployment_virtualenv
+# version of fusionbox-fabric-helpers to run
+HELPERS_COMMIT=f854d6a00
+HELPERS_COMMIT_FILE=$ENV/HELPERS_COMMIT
+
+if [ ! -d $ENV ]
+then
+  virtualenv -p `which python2` $ENV
```

Comment:
Isn't `virtualenv` looking up the path. Isn't `virtualenv -p python2 $ENV` doing the same thing? (This is bike-shedding)